internal/server/server.go

```diff
@@ -77,12 +77,11 @@ func (s *Server) Ping(ctx context.Context, sender *pb.Node) (*pb.Node, error) {
 }
 
 func (s *Server) FindNode(ctx context.Context, request *pb.FindNodeRequest) (*pb.FindNodeResult, error) {
-	// TODO change to func NewKademliaIDFromBytes
-	targetID := (kademliaid.KademliaID)(request.TargetID)
-	senderID := (kademliaid.KademliaID)(request.RequestingNode.ID)
-	candidates := s.routingTable.FindClosestContacts(targetID, env.BucketSize, senderID)
+	targetNodeID := kademliaid.NewKademliaIDFromBytes([20]byte(request.TargetID))
```

Comment:
fixed the todo, but this won't work if `env.BucketSize` is not 20. Should we assume it will always be 20 or try to fix this?

Reply:
can't you use the constant from the env?

Reply:
and don't you mean env.IDLength?